FlowCryptUI/Cell Nodes/TextViewCellNode.swift

```diff
@@ -71,4 +83,9 @@ extension TextViewCellNode: ASEditableTextNodeDelegate {
     public func editableTextNodeDidFinishEditing(_ editableTextNode: ASEditableTextNode) {
         action?(.didEndEditing(editableTextNode.attributedText))
     }
+    
+    public func editableTextNodeDidUpdateText(_ editableTextNode: ASEditableTextNode) {
+        let calculatedHeight = editableTextNode.textView.sizeThatFits(textView.frame.size).height
+        textViewSizeUpdated?(self, calculatedHeight)
```

Comment:
is it possible to resize node inside `TextViewCellNode` without passing this to controller? 
It can avoid calling `setHeight` in case text changed?

``` swift 
public func editableTextNodeDidUpdateText(_ editableTextNode: ASEditableTextNode) {
  let calculatedHeight = editableTextNode.textView.sizeThatFits(textView.frame.size).height
  setHeight(calculatedHeight)   
}
```

Reply:
You can check if `self.supernode` inside `TextViewCellNode` returns node from controller?